core/src/main/java/com/netease/arctic/scan/BaseCombinedScanTask.java

```diff
@@ -50,7 +50,7 @@ public Collection<KeyedTableScanTask> tasks() {
   @Override
   public String toString() {
     return MoreObjects.toStringHelper(this)
-        .add("tasks", Joiner.on(", ").join(tasks))
+        .add("\ntasks", Joiner.on(",\n").join(tasks).replaceAll("\n", "\n\t"))
```

Comment:
Can the statement `Joiner.on(",\n").join(tasks).replaceAll("\n", "\n\t")` be simplified to `Joiner.on(",\n\t").join(tasks)` ?

Reply:
After testing and displaying the logs, I would prefer the prefix format:
```
MoreObjects.toStringHelper(this)
        .add("\ntasks", Joiner.on(",\n").join(tasks).replaceAll("\n", "\n\t"))
        .toString()

BaseCombinedScanTask{
tasks=NodeFileScanTask{
	baseTasks=[], 
	insertTasks=[DefaultKeyedFile{
		file=/tmp/test.parquet, 
		type=I, 
		mask=0, 
		index=0, 
		transactionId=2, 
		fileSizeInBytes=10, 
		recordCount=100}], 
	deleteFiles=[]}}

----
MoreObjects.toStringHelper(this)
        .add("\ntasks", Joiner.on(",\n\t").join(tasks))
        .toString();


BaseCombinedScanTask{
tasks=NodeFileScanTask{
baseTasks=[], 
insertTasks=[DefaultKeyedFile{
	file=/tmp/test.parquet, 
	type=I, 
	mask=0, 
	index=0, 
	transactionId=2, 
	fileSizeInBytes=10, 
	recordCount=100}], 
deleteFiles=[]}}
```